Symfony/CS/Fixer/Contrib/ShortEchoTagFixer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vincent Klaiber <hello@vinkla.com>
+ */
+class ShortEchoTagFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens as $index => $token) {
+            if ('<?=' !== $token->getContent()) {
```

Comment:
Thanks for explaining. Really appreciate it, I want to learn :) 

How about now?

```php
if ('<?=' !== $token->getContent()) {
    continue;
}

$token->override([T_OPEN_TAG, '<?php', $token->getLine()]);

$tokens->insertAt($index, new Token(array(T_ECHO, 'echo')));

if (!$tokens[$index + 1]->isWhitespace()) {
    $tokens->insertAt($index + 1, new Token(array(T_WHITESPACE, ' ')));
}
```